test/district_repository_test.rb

```diff
@@ -1,94 +1,95 @@
 require "minitest/autorun"
 require "minitest/pride"
-require_relative "./../lib/district_repository"
+require "district_repository"
```

Comment:
Try to avoid using require_relative for relative paths (../../) to files that are not in the same directory.